internal/cli/tables-to-go-cli.go

```diff
@@ -45,7 +45,11 @@ func Run(settings *settings.Settings, db database.Database, out output.Writer) (
 		}
 
 		if err = db.GetColumnsOfTable(table); err != nil {
-			return fmt.Errorf("could not get columns of table %s: %v", table.Name, err)
+			if !settings.Force {
+				return fmt.Errorf("could not get columns of table %s: %v", table.Name, err)
+			}
+			fmt.Printf("could not get columns of table %s: %v\n", table.Name, err)
```

Comment:
Just realized while testing that we don't use th `%q` format in these error messages - lets add this in every error message.

Reply:
I changed these two, the other error messages that use strings all seemed to be quoted appropriately.